assets/schema/ensemble_schema.json

```diff
@@ -3513,14 +3513,26 @@
                       "type": "boolean",
                       "description": "If set true, users can view captured/selected media."
                     },
+                    "instantPreview": {
```

Comment:
@snehmehta these seem to be related to the camera and not this ticket

Reply:
yup, it was added by mistake, moving it to it's own PR